### PR TITLE
replace Travis CI with GitHub Actions CI

### PR DESCRIPTION
**Pull request by sullis**
# Motivation

https://arstechnica.com/information-technology/2021/09/travis-ci-flaw-exposed-secrets-for-thousands-of-open-source-projects/

# Changes

- replace Travis CI with GitHub Actions CI workflow

/kind <kind>

Fixes #

**Release Note**

```release-note

```

**Docs**

```docs

```

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    strategy:
+      matrix:
+        java: [8, 11]
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up JDK
+      uses: actions/setup-java@v3
+      with:
+        distribution: 'zulu'
```
Comment (rhuss) [inline .github/workflows/ci.yml:19]: any particular reason why you prefer zulu over temurin ? (the forme adopt openjdk)

**Review by rhuss — COMMENTED** [review]
Thanks, looks good ! I think it makes sense to move over to GithubActions, especially that our build script is really simple.

**Comment by apupier** [comment]
I provided another PR for the same topic which is covering more cases: https://github.com/rhuss/jolokia/pull/522

**Comment by rhuss** [comment]
Sorry folks, for the delay, I'm currently buried by work and it does not look like it will get better until the end of the year;-( 

I we can agree that #522 is more suitable, happy to merge it.

**Comment by rhuss** [comment]
We have merged #522 so we should be good. Closing this PR for now.

**Pull request closed without merging**